test/Dialect/Ibis/round-trip.mlir

```diff
@@ -1,56 +1,54 @@
 // RUN: circt-opt %s | circt-opt | FileCheck %s
 
-// CHECK-LABEL:  ibis.class @A {
+// CHECK-LABEL:  ibis.class @A(%this) {
 // CHECK-NEXT:    ibis.port.input @A_in : i1
 // CHECK-NEXT:    ibis.port.output @A_out : i1
 // CHECK-NEXT:  }
 
-// CHECK-LABEL:  ibis.class @C {
+// CHECK-LABEL:  ibis.class @C(%this) {
 // CHECK-NEXT:    ibis.port.input @C_in : i1
 // CHECK-NEXT:    ibis.port.output @C_out : i1
-// CHECK-NEXT:    ibis.instance @a, @A
-// CHECK-NEXT:    %true = hw.constant true
-// CHECK-NEXT:    ibis.port.write @C_in(%true) : i1
-// CHECK-NEXT:    %0 = ibis.port.read @C_out : i1
-// CHECK-NEXT:    ibis.instance.port.write @a::@A_in(%0) : i1
-// CHECK-NEXT:    %1 = ibis.instance.port.read @a::@A_out : i1
+// CHECK-NEXT:    %0 = ibis.instance @a, @A 
+// CHECK-NEXT:    %1 = ibis.get_parent %0 : !ibis.classref<@A>
+// CHECK-NEXT:    %2 = ibis.get_child @a : @A in %this : !ibis.classref<@C> 
```

Comment:
Plz add a test for getting an instance out of %1 (as will be used with this op).